pkg/apis/config/v1beta1/defaults_test.go

```diff
@@ -0,0 +1,369 @@
+package v1beta1
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/utils/pointer"
```

Comment:
It makes sense.

Reply:
Uhm, we already depend on this package indirectly:
https://github.com/kubeflow/katib/blob/89bd21f710fb4cd153a33c94e8892ec079cf63c8/go.mod#L140

So I'd like to use this library since even if stopping this library, we can not avoid depending on this package.
@andreyvelich  WDYT?

Reply:
@tenzen-y I think, if this package is `indirect` dependency we don't require to manually update the version.
E.g. if `sigs.k8s.io/controller-runtime` requires a newer version of `k8s.io/utils`, it will automatically update the version of this package.
Otherwise, we need to make sure that our version of `k8s.io/utils` is compatible with other packages.
Isn't it ?

Reply:
Yes, that's right.

Reply:
Removed direct dependency on `k8s.io/utils`.